components/zcash_address/src/lib.rs

```diff
@@ -270,6 +272,7 @@ impl ZcashAddress {
                 T::try_from_raw_transparent_p2pkh(data)
             }
             AddressKind::P2sh(data) if regtest_exception => T::try_from_raw_transparent_p2sh(data),
+            AddressKind::Tex(data) if regtest_exception => T::try_from_raw_tex(data),
```

Comment:
Another spot that needs fixing after regtest prefixes were defined:
```suggestion
            AddressKind::Tex(data) if network_matches => T::try_from_raw_tex(data),
```